libraries/botframework-schema/src/ActivityEx.ts

```diff
@@ -0,0 +1,342 @@
+/**
+ * Copyright(c) Microsoft Corporation.All rights reserved.
+ * Licensed under the MIT License.
+ */
+import {
+    IEndOfConversationActivity,
+    IEventActivity,
+    IMessageActivity,
+    IContactRelationUpdateActivity,
+    IConversationUpdateActivity,
+    ITypingActivity,
+    IHandoffActivity,
+    IInvokeActivity,
+    ITraceActivity,
+    IInstallationUpdateActivity,
+    IMessageUpdateActivity,
+    IMessageDeleteActivity,
+    IMessageReactionActivity,
+    ISuggestionActivity } from './activityInterfaces';
+
+import {
+    Activity,
+    ActivityTypes,
+    ChannelAccount,
+    Channels,
+    ConversationReference, 
+    Mention,
+    ResourceResponse} from './index';
+
+export namespace ActivityEx {
+
+    /**
+   * Creates an Activity as an IMessageActivity object.
+   * @returns The new message activity.
+   */
+    export function createMessageActivity(): Partial<IMessageActivity> {
+        return { type: ActivityTypes.Message };
+    }
+
+    /**
+   * Creates an Activity as an IContactRelationUpdateActivity object.
+   * @returns The new contact relation update activity.
+   */
+    export function createContactRelationUpdateActivity(): Partial<IContactRelationUpdateActivity> {
+        return { type: ActivityTypes.ContactRelationUpdate };
+    }
+
+    /**
+   * Creates an Activity as an IConversationUpdateActivity object.
+   * @returns The new conversation update activity.
+   */
+    export function createConversationUpdateActivity(): Partial<IConversationUpdateActivity> {
+        return { type: ActivityTypes.ConversationUpdate };
+    }
+
+    export function createTyingActivity(): Partial<ITypingActivity> {

```

Comment:
Possible typo in method name? `createTyingActivity` > `createTypingActivity`?

Reply:
Feedback applied. Thanks!